Gemfile.lock

```diff
@@ -290,7 +290,7 @@ GEM
     public_suffix (4.0.3)
     puma (4.3.3)
       nio4r (~> 2.0)
-    pundit (1.1.0)
+    pundit (2.1.0)
```

Comment:
if we want >2 for a feature should we be explicit about that in `Gemfile` too?

Reply:
Doesn't hurt